src/Components/Blazor/Blazor/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -31,9 +31,6 @@ public WebAssemblyRenderer(IServiceProvider serviceProvider): base(serviceProvid
             _webAssemblyRendererId = RendererRegistry.Current.Add(this);
         }
 
-        internal void DispatchBrowserEvent(int componentId, int eventHandlerId, UIEventArgs eventArgs)
```

Comment:
Unused

Reply:
Well spotted! It used to be the way that .Browser.JS triggered event handlers, but looks like we moved to a different code path when we redid the JS interop.

Reply:
Lol, I just killed this too as part of my changes